applications/RomApplication/python_scripts/new_python_solvers_wrapper_rom.py

```diff
@@ -51,6 +51,7 @@ def CreateSolverByParameters(model, solver_settings, parallelism, analysis_stage
     #TODO: We could do the same exercise as we do in the stage (module_name to ClassName equal to ModuleName if we standarize the solver names)
     aux_solver_settings = solver_settings.Clone()
     aux_solver_settings.RemoveValue("rom_settings")
+    aux_solver_settings.RemoveValue("solving_strategy")
```

Comment:
If this is the projection strategy, I think it must be encapsulated withing the `rom_settings`.